charts/wharf-helm/values.yaml

```
@@ -101,6 +101,33 @@ web:
   # listens on. [Read more (kubernetes.io/docs)](https://kubernetes.io/docs/concepts/services-networking/service/#defining-a-service)
   containerPort: 8080
 
+  # -- Settings for the OpenId Connect authentication login.
+  oidc:
+
+    # -- The url to the Security Token Service (STS). The authority issues tokens. This field is required.
+    authority: "https://login.microsoftonline.com/841df554-ef9d-48b1-bc6e-44cf8543a8fc/v2.0/.well-known/openid-configuration"
+
+    # -- The Client MUST validate that the aud (audience) Claim contains its client_id value registered at the Issuer identified by the iss (issuer) Claim as an audience. The ID Token MUST be rejected if the ID Token does not list the Client as a valid audience, or if it contains additional audiences not trusted by the Client.
+    clientId: "01fcb3dc-7a2b-4b1c-a7d6-d7033089c779"
+
+    # -- List of scopes which are requested from the server from this client. This must match the Security Token Service configuration for the client you use. The openid scope is required. The offline_access scope can be requested when using refresh tokens but this is optional and some Security Token Service do not support this or recommend not requesting this even when using refresh tokens in the browser.
```

nit: Suggest wrapping these comments on column 80 to get an easier viewing experience on editors that doesn't support wrapping. Such has been attempted in all other comments in this file.

Ex:

```suggestion
    # -- List of scopes which are requested from the server from this client.
    # This must match the Security Token Service configuration for the client
    # you use. The openid scope is required. The offline_access scope can be
    # requested when using refresh tokens but this is optional and some
    # Security Token Service do not support this or recommend not requesting
    # this even when using refresh tokens in the browser.
```

Same applies for all other comments you've added.

---

I really hate these rules that come up ocasionally. Its a linting problem. If it should be adhered to then a simple implementation is sticking `max_line_length` at (i would cast my vote for 100 or 120 ) the intended value in the `.editorconf` file. 

Ideally there would be some consistence across the repos for this aswell, so to me it seems to be a much bigger problem than 1 line in one config file.

Editor conf:
https://github.com/editorconfig/editorconfig/wiki/Property-research%3A-maximum-line-length
Eslint:
https://eslint.org/docs/rules/max-len

I dont have a an tool setup for only refomatting after a particular spec that comes up in github comments. If i reformat with my available tools i catch alot of other inconsistences as well, and wind up with changes across the entire repo.

---

Not everything has to be layed out in config files and be automated. If you really want a computer to tell you this instead of me, then nothing is stopping you from adding an `.editorconfig` file and some CI pipelines to lint this